addons/sale_project/views/project_task_views.xml

```diff
@@ -221,17 +221,17 @@
                        context="{'create': False, 'edit': False, 'delete': False, 'with_price_unit': True}"
                        placeholder="Non-billable"
                        groups="sales_team.group_sale_salesman"
-                       invisible="not allow_billable"/>
-                <field name="sale_line_id" optional="hide" options="{'no_open': True}" readonly="1" groups="!sales_team.group_sale_salesman"/>
+                       invisible="not allow_billable or is_template"/>
+                <field name="sale_line_id" optional="hide" options="{'no_open': True}" readonly="1" groups="!sales_team.group_sale_salesman" invisible="is_template"/>
                 <field name="allow_billable" column_invisible="True"/>
             </xpath>
             <xpath expr="//field[@name='child_ids']/list/field[@name='partner_id']" position="attributes">
                 <attribute name="column_invisible">not parent.allow_billable</attribute>
                 <attribute name="invisible">not allow_billable</attribute>
             </xpath>
             <xpath expr="//field[@name='depend_on_ids']/list/field[@name='partner_id']" position="after">
-                <field name="sale_line_id" optional="hide" readonly="1" groups="sales_team.group_sale_salesman"/>
-                <field name="sale_line_id" optional="hide" options="{'no_open': True}" readonly="1" groups="!sales_team.group_sale_salesman"/>
+                <field name="sale_line_id" optional="hide" readonly="1" groups="sales_team.group_sale_salesman" invisible="is_template"/>
+                <field name="sale_line_id" optional="hide" options="{'no_open': True}" readonly="1" groups="!sales_team.group_sale_salesman" invisible="is_template"/>
```

Comment:
```suggestion
                <field name="sale_line_id" optional="hide" readonly="1" groups="sales_team.group_sale_salesman" column_invisible="parent.is_template"/>
                <field name="sale_line_id" optional="hide" options="{'no_open': True}" readonly="1" groups="!sales_team.group_sale_salesman" column_invisible="parent.is_template"/>
```
? 🤔

Reply:
But it will be more complex than that :cry: